src/main/java/com/hrznstudio/galacticraft/server/command/GalacticraftCommands.java

```diff
@@ -104,4 +117,29 @@ private static int teleportMultiple(CommandContext<ServerCommandSource> context)
         });
         return -1;
     }
+
+    // Feel free to move this method to a utils class. Perhaps this could be cleaned up to return BlockPos.class or Vec3i.class
+    /**
+     * Gets the top y position in the world. Essentially a /top Y coord method
+     * @param world Takes in the world
+     * @param player Takes in the player
+     * @return The top Y value of the world.
+     */
+    private static double getTopBlockY(ServerWorld world, ServerPlayerEntity player) {
+        int playerX = (int) player.getX();
+        int playerZ = (int) player.getZ();
+
+        for (int i = world.getHeight(); i > 0; i-- ) {
+            BlockPos pos = new BlockPos(new Vec3d(playerX, i, playerZ));
+            Block currentBlock = world.getBlockState(pos).getBlock();
+            if (currentBlock != Blocks.VOID_AIR) {
+                if (currentBlock != Blocks.AIR) {
```

Comment:
Maybe use 'Block#isAir' (or is it a block state method?)

Reply:
Oh whoops, meant to keep that on the same line with a || operator. 
VOID_AIR would be any space that's not buildable, at least what I understood.

It works tho, however keeping it on one line could be useful to cleanup.

Reply:
If you're going to still use that, night as well add cave air to the check just in case

Reply:
> If you're going to still use that, might as well add cave air to the check just in case

Marcus suggested using BlockState::isAir, so switched to that.